### PR TITLE
Add and configure iframe resizer

### PR DESCRIPTION
**Pull request by magicznyleszek**
## Description

KPI part of the fix.

## Related issues

Part of #2179

### CODE REVIEW

File: jsapp/js/components/modalForms/projectSettings.es6 [hunk 1]
```diff
@@ -751,6 +751,10 @@ class ProjectSettings extends React.Component {
     const sectors = session.currentAccount.available_sectors;
     const countries = session.currentAccount.available_countries;
 
+    if (this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING && this.props.iframeUrl) {
+      $('iframe#kc-settings-iframe').iFrameResize({log: true});
```
Comment (Eric-Dilcher) [inline jsapp/js/components/modalForms/projectSettings.es6]: the selector is unnecessarily specific.  Anyways, IDs are unique across applications.

File: jsapp/js/components/modalForms/projectSettings.es6 [hunk 2]
```diff
@@ -869,7 +873,12 @@ class ProjectSettings extends React.Component {
 
           {this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING && this.props.iframeUrl &&
             <bem.FormView__cell m='iframe'>
-              <iframe src={this.props.iframeUrl} />
+              <iframe
+                id='kc-settings-iframe'
+                src={this.props.iframeUrl}
+                scrolling='no'
+                style={{width: '1px', minWidth: '100%'}}
```
Comment (Eric-Dilcher) [inline jsapp/js/components/modalForms/projectSettings.es6]: why do you have this style?  Why isn't it in css?   I'm assuming it's needed for iframe-resizer to work.  Please add a comment that says that

File: package.json [hunk 3]
```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-react": "^7.10.0",
     "file-loader": "^2.0.0",
     "font-awesome": "^4.4.0",
+    "iframe-resizer": "^3.6.5",
```
Comment (Eric-Dilcher) [inline package.json]: looks like v4 was published not that long ago.  You should update to it :)

File: jsapp/js/components/modalForms/projectSettings.es6 (hunk 2 above)
Comment (Eric-Dilcher) [reply jsapp/js/components/modalForms/projectSettings.es6]: OK, I see the note in the package.

> Note: Using min-width to set the width of the iFrame, works around an issue in iOS that can prevent the iFrame from sizing correctly.

Can you add this note to the css file when you move the style there?

**Comment by jnm** [comment]
I can't make it work locally :sob: 
cry

Ah yes, if only I had read the console messages:
`Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('http://172.17.0.1:8001') does not match the recipient window's origin ('http://172.17.0.1:8000').`
image

File: jsapp/js/components/modalForms/projectSettings.es6 (hunk 1 above)
Comment (magicznyleszek) [reply jsapp/js/components/modalForms/projectSettings.es6]: fixed :)

File: jsapp/js/components/modalForms/projectSettings.es6 (hunk 2 above)
Comment (magicznyleszek) [reply jsapp/js/components/modalForms/projectSettings.es6]: fixed :)

File: package.json (hunk 3 above)
Comment (magicznyleszek) [reply package.json]: updated here and on the other branch

**Comment by magicznyleszek** [comment]
@jnm any idea how to test it locally? Can we deploy this branch to a test server and the `kobocat-template` branch too?

**Comment by magicznyleszek** [comment]
Close in favor of #2238

**Pull request closed without merging**